hugegraph-core/src/main/java/com/baidu/hugegraph/backend/tx/GraphIndexTransaction.java

```diff
@@ -153,6 +156,17 @@ public void updateLabelIndex(HugeElement element, boolean removed) {
 
     @Watched(prefix = "index")
     public void updateVertexIndex(HugeVertex vertex, boolean removed) {
+        if (vertex.olap()) {
+            Id pkId = vertex.getProperties().keySet().iterator().next();
```

Comment:
only one pkId?

Reply:
yes. write back only one olap property for one vertex id